additionalCSS/footerSection.css

```diff
@@ -16,3 +16,151 @@
   cursor: pointer;
   background-color: #008552;
 }
+
+.contactForm {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  /* justify-self: center; */
+
+  /* width: 90%; */
+
+  /* margin: auto; */
+  padding: 0 24px;
+
+  /* text-align: center; */
+}
+
+.btn-center {
+  display: flex;
+  justify-content: center;
+}
+
+.fText {
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #6b778c;
+  font-style: normal;
+  padding: 15px;
+}
+
+:hover::placeholder {
+  transform: translate(10px);
+}
+
+form input {
+  /* width: 327px; */
+  height: 48px;
+  border-radius: 4px;
+  border: 1px solid #d0d9d4;
+  background: #fff;
+}
+
+.fEmail {
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  margin-top: 24px;
+  color: #6b778c;
+  font-style: normal;
+  padding: 15px;
+}
+
+form textarea {
+  /* width: 327px; */
+  height: 114px;
+  border-radius: 4px;
+  border: 1px solid #60c095;
+  margin-top: 24px;
+  background: #fff;
+}
+
+.fTextArea {
+  font-family: "Inter", sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  margin-top: 24px;
+  color: #091e42;
+  font-style: normal;
+  padding-left: 16px;
+  padding-top: 12px;
+}
+
+.fTextArea::placeholder {
+  color: black;
+}
+
+.footerBtn {
+  text-align: center;
+}
+
+.getInTouch {
+  padding: 12px;
+  background: #36b37f;
+  font-family: "Inter", sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  border-style: none;
+  border-radius: 4px;
+  margin-top: 24px;
+  text-align: center;
+
+  /* margin-left: 99px; */
```

Comment:
Kindly take away every commented block of code from your project😉 It is best practice to always remove the commented blocks of code since having them in your code base make it look messy and less professional👍😉 This is also present some other times in most of your style sheet😉

Reply:
Hello @Ibizugbe. Thank you for your kind feedback, I'll have that fixed.